sequence_processing_pipeline/FastQCJob.py

```diff
@@ -10,53 +10,45 @@
 class FastQCJob(Job):
     def __init__(self, run_dir, output_directory, nprocs, nthreads,
                  fastqc_path, modules_to_load, qiita_job_id, run_id,
-                 queue_name, node_count, wall_time_limit, pmem):
+                 queue_name, node_count, wall_time_limit, jmem, pool_size):
         self.job_name = 'FastQCJob'
         super().__init__(run_dir, self.job_name, [fastqc_path],
                          modules_to_load)
         self.nprocs = nprocs
         self.nthreads = nthreads
         self.fastqc_path = fastqc_path
         self.modules_to_load = modules_to_load
-        # assume that the path to raw fastqs is run_dir/Data/Fastq
         self.raw_fastq_path = join(self.run_dir, 'Data', 'Fastq')
-        # output_directory is the root path for creating a FastQC directory
-        # with FastQC output. It's also the location for finding trimmed and
-        # filtered fastq files as well.
         self.fastqc_output_path = output_directory
-        # '.../output_dir/run_id/project/filter_type'
         self.processed_fastq_path = join(self.run_dir, run_id)
         self.qiita_job_id = qiita_job_id
         self.trim_file = 'fqc_split_file_'
-
         self.queue_name = queue_name
         self.node_count = node_count
         self.wall_time_limit = wall_time_limit
         self.products_dir = join(self.run_dir, run_id)
-        self.pmem = pmem
+        self.jmem = jmem
+        self.pool_size = pool_size
 
-        self.commands = self.get_cmds()
-        split_count = self._generate_split_count(len(self.commands))
-        lines_per_split = int(
-            (len(self.commands) + split_count - 1) / split_count)
-        self._generate_trim_files(self.commands, lines_per_split)
+        self.cmds = self._get_cmds()
```

Comment:
Not super important right now but wanted to highlight that we prefer to be clear with the variables being used, an example commands vs. cmds; perhaps worth taking a look at http://fperez.org/py4science/code_reviews.html.